Themis/Supporting/ThemisUndoManager.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ThemisUndoManager.swift
+//  Themis
+//
+//  Created by Tarlan Ismayilsoy on 24.05.23.
+//
+
+import Foundation
+
+/// A wrapper class intended to perform UndoManager actions and publish notifications to update views
+class ThemisUndoManager {
```

Comment:
Is there a specific reason why you chose delegation over inheritance here? (I'm happy with it, just curious)